src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -330,6 +354,84 @@ public Response getIterationDetails(
         return Response.ok(gtfsRtFeedIterationModel).build();
     }
 
+    // Returns past session details for a particular clientId.
+    @GET
+    @Path("/{clientId : .*}/pastSessions")
```

Comment:
IMHO client ID would be better as a query parameter here because you aren't performing an operation on that client - you're asking to retrieve information about a particular client (I.E., a query).  But let me know if I'm missing something.